Xamarin.Forms.Platform.Android/CollectionView/EmptyViewAdapter.cs

```diff
@@ -47,14 +47,15 @@ public override RecyclerView.ViewHolder OnCreateViewHolder(ViewGroup parent, int
 				}
 
 				// EmptyView is a Forms View; display that
-				var itemContentControl = new SizedItemContentControl(CreateRenderer(formsView, context), context,
+				var itemContentControl = new SizedItemContentView(CreateRenderer(formsView, context), context,
```

Comment:
Why was this type renamed?

Reply:
The `ItemContentControl` naming was a carryover from the UWP implementation. I changed it to make more sense in the context of Android.